scopes/react/component-highlighter/element-highlighter/element-highlighter.module.scss

```diff
@@ -6,3 +8,8 @@
   font-family: sans-serif;
   padding: 8px;
 }
+
+.frame,
+.label {
+  z-index: $highlighter-z-index;
```

Comment:
can we use as a css var so we can override if needed?
 `var(--bit-highlighter-zindex,  $highlighter-z-index);`